CHANGELOG

```diff
@@ -1,4 +1,5 @@
 == X.Y.Z 2017-XX-YY
+* Many methods on `STPCard` and `STPCardParams` have been deprecated or made readonly to bring card objects more in line with the rest of the API. See MIGRATING for further details.
```

Comment:
Just starting a discussion, what do we think about starting all of our changelog lines with "Adds / Improves / Fixes"? In this case, I think it would be more like "Fixes many methods..."